harvester/conftest.py

```diff
@@ -0,0 +1,100 @@
+"""This is a gathering of fixtures."""
```

Comment:
We should use the same naming schema for test files. Sincve I started (:P) I can choose that we start with `test_`. Please rename `conftest.py` to `test_conf.py`